solarforecastarbiter/metrics/event.py

```diff
@@ -0,0 +1,232 @@
+"""Event forecast error metrics."""
+
+import numpy as np
+
+
+def _event2count(obs, fx):
+    """Convert events (True/False) into counts.
+
+    Given forecasts and observations of events (True=event occurred,
+    False=event did not occur), the pairs of forecasts and observations can be
+    placed into four categories:
+    - True Positive (TP): forecast = event, observed = event
+    - False Positive (FP): forecast = event, observed = no event
+    - True Negative (TN): forecast = no event, observed = no event
+    - False Negative (FN): forecast = no event, observed = event
+
+    Parameters
+    ----------
+    obs : (n,) array-like
+        Observed event values (True=event, False=no event).
+    fx : (n,) array-like
+        Forecasted event values (True=event, False=no event).
+
+    Returns
+    -------
+    tp : int
+        Number of true positives.
+    fp : int
+        Number of false positives.
+    tn : int
+        Number of true negatives.
+    fn : int
+        Number of false negatives.
+
+    Raises
```

Comment:
all the other functions that rely on this should also have this raises section

Reply:
Just pushed a commit adding the Raises section to the docstring for all functions that call `_event2count()` (as suggested).